src/app/pages/data-protection/replication/replication-wizard/replication-wizard.component.ts

```diff
@@ -888,11 +892,10 @@ export class ReplicationWizardComponent implements WizardConfiguration {
 
     const sshCredentialsSourceField = _.find(this.sourceFieldSet.config, { name: 'ssh_credentials_source' }) as FormSelectConfig;
     const sshCredentialsTargetField = _.find(this.targetFieldSet.config, { name: 'ssh_credentials_target' }) as FormSelectConfig;
-    this.keychainCredentialService.getSshConnections().pipe(untilDestroyed(this)).subscribe((connections) => {
-      connections.forEach((connection) => {
-        sshCredentialsSourceField.options.push({ label: connection.name, value: connection.id });
-        sshCredentialsTargetField.options.push({ label: connection.name, value: connection.id });
-      });
+    this.keychainCredentialService.getSshConnections().pipe(untilDestroyed(this)).subscribe((credentials) => {
+      this.sshCredentials = credentials.map((credential) => ({ label: credential.name, value: credential.id }));
+      sshCredentialsSourceField.options = [...this.sshCredentials];
+      sshCredentialsTargetField.options = [...this.sshCredentials];
```

Comment:
Duplicated lines.

Reply:
These are different constants. Above is the definition of these.
```    
 const sshCredentialsSourceField = _.find(this.sourceFieldSet.config, { name: 'ssh_credentials_source' }) as FormSelectConfig;
 const sshCredentialsTargetField = _.find(this.targetFieldSet.config, { name: 'ssh_credentials_target' }) as FormSelectConfig;
```